app/connectors/AssociationConnector.scala

```diff
@@ -70,12 +71,10 @@ class AssociationConnectorImpl @Inject()(
                                  headerCarrier: HeaderCarrier,
                                  ec: ExecutionContext,
                                  request: RequestHeader): Future[Either[HttpException, MinimalDetails]] = {
-    implicit val hc: HeaderCarrier = HeaderCarrier(extraHeaders =
-      headerUtils.integrationFrameworkHeader)
-    val minimalDetailsUrl = appConfig.psaMinimalDetailsUrl.format(regime, idType, idValue)
-    httpClient.GET(minimalDetailsUrl)(implicitly[HttpReads[HttpResponse]], implicitly[HeaderCarrier](hc),
-      implicitly) map {
-      handleResponseIF(_, minimalDetailsUrl)
+
+    val minimalDetailsUrl = url"${appConfig.psaMinimalDetailsUrl.format(regime, idType, idValue)}"
+    httpV2Client.get(minimalDetailsUrl).setHeader(headerUtils.integrationFrameworkHeader: _*).execute[HttpResponse] map {
+      handleResponseIF(_, minimalDetailsUrl.toString)
```

Comment:
Does this throw a warning? Something like http reads raw?
In instances where HttpResponse is used, need to add the following import:
import uk.gov.hmrc.http.HttpReads.Implicits._